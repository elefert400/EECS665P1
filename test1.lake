+-
+1
while
&&
return
5_ab15c_
read
"ab\nc"
"\n"
#yellow
"include a quote like this \" and a backslash like this \\"
