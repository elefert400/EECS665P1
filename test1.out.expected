+
-
+
INTLIT:1
EOF
